main/settings.py

```diff
@@ -112,16 +112,30 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': "osp",
-        'USER': "osp",
-        'PASSWORD': "osp",
-        'HOST': "localhost",
-        'PORT': 5432,
+if os.environ.get("GITHUB_WORKFLOW"):
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.db.backends.postgresql_psycopg2",
+            "NAME": "postgres",
+            "USER": "postgres",
+            "PASSWORD": "osp",
+            "HOST": "localhost",
+            "PORT": 5432,
+        }
     }
-}
+
+else:
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.db.backends.postgresql_psycopg2",
+            "NAME": "osp",
+            "USER": "osp",
+            "PASSWORD": "osp",
+            "HOST": "localhost",
+            "PORT": 5432,
+        }
+    }
+
```

Comment:
Don't make changes here. Instead use these variables in the workflow itself

Reply:
I followed GitHub action docs to create this container. Is creating env variables for this work?